src/frequenz/client/dispatch/__main__.py

```diff
@@ -0,0 +1,211 @@
+# License: All rights reserved
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""CLI and interactive client for the dispatch service."""
+
+import asyncio
+import os
+import sys
+from datetime import datetime, timedelta, timezone
+from typing import Any, List, cast
+
+import asyncclick as click
+import grpc
+from prompt_toolkit import PromptSession
+from prompt_toolkit.completion import NestedCompleter
+from prompt_toolkit.history import FileHistory
+from prompt_toolkit.patch_stdout import patch_stdout
+from prompt_toolkit.shortcuts import CompleteStyle
+
+from frequenz.client.common.microgrid.components import ComponentCategory
+from frequenz.client.dispatch import Client
+from frequenz.client.dispatch.types import ComponentSelector
+
+DISPATCH_API_HOST = os.getenv("DISPATCH_API_HOST", "88.99.25.81")
+"""Dispatch API Host
+
+Taken from the DISPATCH_API_HOST environment variable.
+"""
+
+DISPATCH_API_PORT = int(os.getenv("DISPATCH_API_PORT", "50051"))
+"""Dispatch API Port
+
+Taken from the DISPATCH_API_PORT environment variable.
+"""
```

Comment:
There is [built-in support in `click` for environment variables](https://click.palletsprojects.com/en/8.1.x/options/#values-from-environment-variables) (and since `asyncclick` points to `click` docs I assume in `asyncclick` too). You might want to use that instead.